docs/dev/plugins/cloudformation/cloudformation-basics.md

```diff
@@ -0,0 +1,37 @@
+# AWS CloudFormation
+How to transform a TOSCA model to an Amazon Web Services(AWS) CloudFormation template.
+
+## Mapping
+Mappings of TOSCA node types to AWS Resources are shown in [this table](mapping.md).
+
+## Transformation Lyfecycle
+
+### validate
+- Nothing to do here so far
```

Comment:
Please change this to TBD at this point.

This sounds like your Plugin will not need a Validation phase, meaning it can transform everthing.

Also consider the to phases you have to implement yourself:
- Environment validation 
- Model validation

Read more [here](https://github.com/StuPro-TOSCAna/TOSCAna/blob/master/docs/dev/plugins/transformation-lifecycle/lifecycle-architecture.md#validation-phase)